### ISSUE TITLE
change dataset landing page elements

### ISSUE THREAD

**Issue by satra** [dandi/dandiarchive#651]
if you scroll down to the bottom the name of the project looks really weird. https://gui.dandiarchive.org/#/dandiset/000026

![image](https://user-images.githubusercontent.com/184063/117151827-6d3f7e80-ad87-11eb-8f99-3dfb226944dc.png)

**Comment by satra** [comment]
more generally i think we would want to move quickly on improving the elements of the landing page now that the schema is somewhat reasonable.

**Comment by waxlamp** [comment]
The [schema section](https://github.com/dandi/schema/blob/master/releases/0.3.0/dandiset.json#L872-L878) for the `name` property is an array of values. I believe that's what's making the render look weird.

Is it supposed to be an array? If so, why?

If yes, what should we do to improve the rendering for the name?

**Comment by satra** [comment]
a dandiset can be associated with multiple projects. so this should simply show the names of those projects for now.

**Comment by waxlamp** [comment]
@AlmightyYakob says he can address this one on Monday.

@satra, the issue title seems more general than the description. Do you have more landing page elements that need to be updated?

**Comment by AlmightyYakob** [comment]
@satra also FWIW I think the title of the `wasGeneratedBy` schema field (this field) should be changed to `Projects` or something, to more accurately convey the intent, as `Name of the project` makes it seem like there's only one.

**Comment by satra** [comment]
One of the issues is how things would be reflected when people start adding new things to the metadata. We found this only after adding the field, hence the specific issue.

Other things that need to change are things like removing schemaKey. 

> should be changed to Projects 

We can update that in the schema.

More generally, I think the landing page should be a curated view of the Dandiset metadata. I.e. given the model and the info on the dandiset we should say this is what we want to show, and these are the visual transforms from the metadata to the render.  We should perhaps revisit this landing page in a brainstorming session.

**Comment by waxlamp** [comment]
> More generally, I think the landing page should be a curated view of the Dandiset metadata. I.e. given the model and the info on the dandiset we should say this is what we want to show, and these are the visual transforms from the metadata to the render. We should perhaps revisit this landing page in a brainstorming session.

:100:

I agree that the landing page should not just be a dynamic view of a bunch of metadata items. To start, can you identify some metadata records that should be part of this curated view? Then we can at least begin to get the look and feel of that starting subset right.

I think:
- dandiset ID
- name
- description
- contributors
- owners
- related projects

are in the list. Any others?

**Comment by waxlamp** [comment]
I'll bring in Jared or Faiza to help with the UI/UX design work.

**Comment by waxlamp** [comment]
We need to transplant the larger discussion of revamping the landing page into a new issue.
